hoomd/md/methods/thermostats.py

```diff
@@ -248,16 +248,38 @@ class Bussi(Thermostat):
         kT (hoomd.variant.variant_like): Temperature set point for the
             thermostat :math:`[\mathrm{energy}]`.
 
-    Provides temperature control by rescaling the velocity by a factor taken
-    from the canonical velocity distribution. On each timestep, velocities are
-    rescaled by a factor :math:`\alpha=\sqrt{K_t / K}`, where :math:`K` is the
-    current kinetic energy, and :math:`K_t` is chosen randomly from the
-    distribution
+        tau (float): Thermostat time constant :math:`[\mathrm{time}]`.
+            Defaults to 0.
+
+    `Bussi` controls the system temperature by separately rescaling the velocity
+    and angular momenta by the factor :math:`\alpha` sampled from the canonical
+    distribution.
+
+    When `tau` is 0, the stochastic evolution of system is instantly thermalized
+    and :math:`\alpha` is given by:
+
+    .. math::
+        \alpha = \sqrt{\frac{g_N kT}{K}}
+
+    where :math:`K` is the instantaneous kinetic energy of the corresponding
+    translational or rotational degrees of freedom, :math:`N` is the number of
+    degrees of freedom, and :math:`g_N` is a random value sampled from the
+    gamma distribution :math:`\mathrm{Gamma}(N, 1)`:
 
     .. math::
-        P(K_t) \propto K_t^{N_f/2 - 1} \exp(-K_t / kT)
+        f_N(g) = \frac{1}{\Gamma(N)} g^{N-1} e^{-g}.
 
-    where :math:`N_f` is the number of degrees of freedom thermalized.
+    When `tau` is non-zero, the kinetic energies decay to equilibrium with the
+    given characteristic time constant and :math:`\alpha` is given by:
+
+    .. math::
+        \alpha = \sqrt{e^{\delta t / \tau}
+                 + (1 - e^{\delta t / \tau}) \frac{(2 g_{N-1} + n^2) kT}{2 K}
+                 + 2 n \sqrt{e^{\delta t / \tau} (1-e{\delta t / \tau})
```

Comment:
I believe there is a typo
```suggestion
                 + 2 n \sqrt{e^{\delta t / \tau} (1-e^{\delta t / \tau})
```